### PR TITLE
Feature/#11 rewrite elementFactory

### LINKED ISSUE #11: Feature/#11 Rewrite elementFactory from StarWars project into TypeScript

### PR DESCRIPTION
**Pull request by SebastianBabinski1**
closes #11

### CODE REVIEW

**Review by Enessetere — APPROVED** [review]
Looks good to me, well done.

File: src/app/App.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-export class App {
-  init() {
+class App {
+  _init() {
     return 'hello world';
   }
 }
+
+export default App;
```
Comment (DominikNowak) [inline src/app/App.ts]: Changes for this file are reduntant.

File: src/index.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { App } from './app/App';
+import App from './app/App';
 
 window.onload = () => {
-  new App().init();
+  new App()._init();
```
Comment (DominikNowak) [inline src/index.ts]: So in this one too

**Review by DominikNowak — APPROVED** [review]
Well done :)

**Pull request merged**